src/modules/item/export.ts

```diff
@@ -41,9 +42,9 @@ async function getFiles(contents: Record<string, string>) {
   }, {})
 }
 
-function toCatalystItem(collection: Collection, item: Item): CatalystItem {
+function toCatalystItem(collection: Collection, item: Item, chainId: ChainId): CatalystItem {
   return {
-    id: item.id,
+    id: getCatalystURN(collection, item, chainId),
```

Comment:
I know that we have only collections right now, but maybe we can call it `getCatalystCollectionItemURN` or `getCatalystItemURN`. What do you think?

Reply:
Oh good point, didn't realize it was scoped by collection